variables.tf

```diff
@@ -636,3 +636,51 @@ variable "horizon-plugins" {
   type        = list(string)
   default     = []
 }
+
+variable "enable-validation" {
+  description = "Enable Tempest deployment"
+  type        = bool
+  default     = false
+}
+
+variable "tempest-channel" {
+  description = "Operator channel for Tempest deployment"
+  type        = string
+  default     = null
+}
+
+variable "tempest-revision" {
+  description = "Operator channel revision for Tempest deployment"
+  type        = number
+  default     = null
+}
+
+variable "tempest-config" {
+  description = "Operator config for Tempest deployment"
+  type        = map(string)
+  default     = {}
+}
+
+variable "grafana-agent-k8s-name" {
```

Comment:
Grafana agent k8s and tempest-k8s will be on the same model: the openstack model (i.e. the control plane). But they will be deployed using different terraform plans. In the terraform plan with tempest-k8s, we don't have access to the name of the other juju resource, so we need to pass the name here.

Reply:
Link you provided is not working.
I know that tempest-k8s and grafana-agent-k8s will be on same model, but I thought that it will be separate model, not the control plane one. So now I want to understand why we need/want to have different name for grafana-agent-k8s. We will use same (for multiple apps) grafana-agent? Or we will deploy one only for tempest-k8s?

Reply:
We could hardcode the name here, and make sure the name defined in observability plugin matches the name defined here. But there will be two degree-of-freedoms: (1) grafana-agent-k8s name defined here, and (2) grafana-agent-k8s name defined in observability plugin, and (1) and (2) must match.

Reply:
But yes, my argument here is quite weak. We could hardcode the name if you prefer

Reply:
Ok, so they are or two different repos. Now it makes sense, but it's still have felling that it's strange.